Shared/src/Datahub.Shared/Exceptions/ProjectNotFoundException.cs

```diff
@@ -0,0 +1,10 @@
+using System;
+
+namespace Datahub.Shared.Exceptions;
+
+public class ProjectNotFoundException : Exception
+{
+    public ProjectNotFoundException(string message) : base(message)
+    {
+    }
+}
```

Comment:
![image](https://user-images.githubusercontent.com/43834308/212698944-c030eccd-f044-4d08-a90a-1b2a04609a64.png)
Hahaha, can we keep just the one?